setup.py

```diff
@@ -14,8 +14,8 @@
 from numpy.distutils.command.build_ext import build_ext
 
 
-DISTNAME = 'scikit-image'
-DESCRIPTION = 'Image processing in Python'
+DISTNAME = 'scikit-image-testviniver'
+DESCRIPTION = 'Just a test'
```

Comment:
:confused:

Reply:
@leej3 ☝️ this is the cause of the appveyor failure, should go away after that is removed.

I suggest simply removing those lines and we can squash and merge the PR here — no need to manually squash the commits.